src/Time/Formatting.hs

```diff
@@ -30,13 +30,15 @@ __Examples__
 The received list should be in descending order. It would be verified at compile-time.
 Example of the error from @ghci@:
 
->>> seriesF @'[Millisecond, Second] (minute 42)
+@
+__>>> seriesF @'[Millisecond, Second] (minute 42)__
```

Comment:
Could you implement this as `doctest` test as well? Just see how I did here in `universum`:

* https://github.com/serokell/universum/blob/68727edc7c1c478a5de09677b2ab0b1e846f7615/src/Universum/Container/Class.hs#L604

You can use `...` to skip uninteresting parts. I think we should test this...